heron/api/tests/java/BUILD

```diff
@@ -0,0 +1,26 @@
+load("/tools/rules/java_tests", "java_tests")
+load("/tools/rules/heron_deps", "heron_java_proto_files")
+
+api_deps_files = [
+    "//heron/api/src/java:api-java",
+    "//third_party/java:junit4",
+]
+
+api_deps_files = \
+  heron_java_proto_files() + \
```

Comment:
can we have heron_java_api_proto_files instead

Reply:
sure